public/locales/en/translation.json

```diff
@@ -140,7 +140,8 @@
               "error": {
                 "label" : "Error",
                 "empty" : "Scenario name cannot be empty",
-                "existing": "Scenario name already exists"
+                "existing": "Scenario name already exists",
+                "forbiddenchars": "Scenario name has to starts with a letter, and can only contains letters, digits, spaces, underscores and hyphens."
```

Comment:
Small typos: no 's' in "starts" and "contains"
"Scenario name has to start with a letter, and can only contain letters, digits, spaces, underscores and hyphens"